desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -239,10 +239,8 @@ def execute(self, notebook, snippet):
 
     engine = self._get_engine()
     connection = self._create_connection(engine)
-    statement = snippet['statement']
-
-    if self.interpreter['dialect_properties'].get('trim_statement_semicolon', True):
-      statement = statement.strip().rstrip(';')
```

Comment:
You just removed the `trim_statement_semicolon` property? (PS: if `get_statements()` is good enough to apply for every dialect, we should fully remove the property throughout all the code base, not just in one spot)